src/components/common/RiskChart/index.tsx

```diff
@@ -0,0 +1,287 @@
+/* eslint-disable react/display-name */
+import { GenericList, GenericListItem } from '../../app';
+import { CellPosition, HeadCell } from '../../app/GenericList/HeadCell';
+import { colors, getSemaphoreInfo } from '../Semaphore';
+
+const splitString = (
+    item: string,
+    separator: string,
+    defaultList: string[]
+) => {
+    return item === '' ? defaultList : item.split(separator);
+};
+
+export const headCells: HeadCell[] = [
+    {
+        id: 'label',
+        numeric: true,
+        disablePadding: false,
+        label: 'Impact',
+        align: 'center',
+    },
+    {
+        numeric: true,
+        disablePadding: false,
+        label: 'Rare (1)',
+        align: 'center',
+        format: (
+            item: GenericListItem,
```

Comment:
possible next version we can have a link constructed by the group name
/grouping/default/group/:groupLabel

That loads the query page data detail of contracts belonging to the group